usr/sbin/p11sak/p11tool.c

```diff
@@ -0,0 +1,2377 @@
+/*
+ * COPYRIGHT (c) International Business Machines Corp. 2001-2025
```

Comment:
as this is a new source, just 2025? Where does the 2001 come from?

Reply:
Well, because the code was basically just moved to this new file, so I thought I keep the from date. But I am not sure if this is 'correct'....